http/headers/Sec-Purpose.json

```diff
@@ -0,0 +1,44 @@
+{
+  "http": {
+    "headers": {
+      "Sec-Purpose": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Sec-Purpose",
+          "spec_url": "https://fetch.spec.whatwg.org/#sec-purpose-header",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "notes": "Chrome uses the legacy <code>Purpose: prefetch</code> header to indicate a request is a prefetch. See <a href='https://crbug.com/1358419'>bug 1358419</a>."
```

Comment:
This might perhaps be considered an Alias. Thing is, I don't know how they plan on deprecating or when the `Purpose` header was added.